frontend/src/slices/auth/auth.slice.ts

```diff
@@ -27,6 +30,16 @@ const { reducer, actions, name } = createSlice({
     builder.addCase(signUp.rejected, (state) => {
       state.dataStatus = DataStatus.REJECTED;
     });
+    builder.addCase(getUser.pending, (state) => {
+      state.dataStatus = DataStatus.PENDING;
```

Comment:
for authentification request let's use another property - authenticatedUserDataStatus